graalvm/java11/cookiecutter-aws-sam-graalvm-maven/{{cookiecutter.project_name}}/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM public.ecr.aws/amazonlinux/amazonlinux:2
+
+RUN yum -y update \
+    && yum install -y unzip tar gzip bzip2-devel ed gcc gcc-c++ gcc-gfortran \
+    less libcurl-devel openssl openssl-devel readline-devel xz-devel \
+    zlib-devel glibc-static libcxx libcxx-devel llvm-toolset-7 zlib-static \
+    && rm -rf /var/cache/yum
+
+# Graal VM
+ENV JAVA_VERSION java11
+ENV GRAAL_VERSION 22.0.0.2
+ENV GRAAL_FOLDERNAME graalvm-ce-${JAVA_VERSION}-${GRAAL_VERSION}
+ENV GRAAL_FILENAME graalvm-ce-${JAVA_VERSION}-linux-amd64-${GRAAL_VERSION}.tar.gz
+RUN curl -4 -L https://github.com/graalvm/graalvm-ce-builds/releases/download/vm-${GRAAL_VERSION}/${GRAAL_FILENAME} | tar -xvz
+RUN mv $GRAAL_FOLDERNAME /usr/lib/graalvm
+RUN rm -rf $GRAAL_FOLDERNAME
+
+# Maven
+ENV MVN_VERSION 3.8.4
```

Comment:
Seems like maven version 3.8.4 is not available anymore but 3.8.5 is. Can we update with that?